_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,83 @@
+---
+title: What's the point of a simple theory?
+subtitle: A story about the power of null models to shape how we interpret data
+#status: inactive #active if you want it on the frontpage
+
+category: paper
+
+description: |
+  A story about the power of null models to shape how we interpret data.
+
+people: # add people that are involved in this project
+  - maddy
+
+layout: blog # do not change this
+image: '/img/posts/why-null-models-crispr/banfield_data_theory.svg'
+#link:
+no-link: false # if you don't want it's own webpage
+last-updated: 2022-01-08
+# do NOT add 0 (such as 06 for June). Will break everything lol
+# take out the underscore in the name.
+---
+
+## Why do we need theory?
+
+Experimentalists wade through the murky waters of biology, hoping to tease out knowledge by controlling the things they can control and carefully observing what they cannot. In a well-designed experiment, the observed data reveals some truth, and this truth is reached by painstakingly ruling out other explanations. This might sound logical and sytematic, but it's an art form: how do you know when you're done considering reasonable alternatives? And in biology, where some [truly unreasonable](https://www.theatlantic.com/science/archive/2017/06/jumping-spiders-can-see-the-moon/529329/) things can happen, how do you know what's 'reasonable' anyway?
+
+This is where theory can help, and in particular, theory so simple that it predicts what would happen if *nothing* weird or cool was happening. Theories like this are often called ''null models'' because they model the ''null'' (''zero'' or ''nothing'') expectation. And if your null model ends up doing something weird or cool after all? Well, you're in business! 
```

Comment:
I got a little confused here. First paragraph I got: _experiment -> data -> possible explanation, but are there other possible explanations?_ And then here it jumps to: _null models can help_. But why exactly? From what I understand, the goal here is to find other possible explanations for a dataset (or to rule them out), and a first line of investigation would be to try out a null hypothesis which can lead to a prediction using theory. I think something should be added here to clear this up a little.

Reply:
Thanks! Made some edits to the intro to be more clear.